ckanext/dcatapit/commands/dcatapit.py

```diff
@@ -38,6 +39,22 @@ class DCATAPITCommands(CkanCommand):
           frequencies -> http://publications.europa.eu/mdr/resource/authority/frequency/skos/frequencies-skos.rdf
           filetype -> http://publications.europa.eu/mdr/resource/authority/file-type/skos/filetypes-skos.rdf
        PATH_TO_INI_FILE is the path to the Ckan configuration file
+
+
+     # load theme to groups mapping, and recreate existing mappings
+     paster --plugin=ckanext-dcatapit vocabulary import_theme_mapping FILE
+
+     Where:
+        FILE is a path to configuration file
+
+
+     Configuration file format: .ini:
+
+[dcatapit:theme_group_mapping]
+NAME_OF_THEM = list,of,groups,
+    separated by newline
+    or, by coma
+
```

Comment:
Since we are no longer importing the mapping into the DB, this description should be removed.